packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -482,6 +494,20 @@ contract Deploy is Deployer {
         addr_ = address(proxy);
     }
 
```

Comment:
We will likely want to update the `SystemConfig` to have a pointer to the dac contract, the latest release of the `SystemConfig` has pointers to the proxies for contracts specific to a particular op stack chain instance

Reply:
updated